### PR TITLE
Deploy funny things application

### PR DESCRIPTION
**Pull request by lauracavaliere**
In this PR we implemented the deployment for the funny things application. In particular:
- we generated the `.yml` files through our convert starting from an `.XML` file of the application
- we created the GUI where the user has to insert the `.json` file, mandatory to use the google speech service in the application, and he can select the language and voice as desired.

### CODE REVIEW

File: demos/funnyThingsApp/composeHead.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - "/dev:/dev"
+  ports:
+    - "10000:10000"
+  network_mode: "host"
+  privileged: true
+
+services:
+  yFunnyThingsApp:
+    <<: *yarp-base
+    command: ls -la
```
Comment (AlexAntn) [inline demos/funnyThingsApp/composeHead.yml:19]: Do we need this file? it is not doing anything in the deployment as it is

Comment (vtikha) [reply demos/funnyThingsApp/composeHead.yml:19]: no we do not

File: demos/funnyThingsApp/gui/gui_conf.ini [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title "Funny Things App" 
+
+[top options] 
+ImageName "images/icub-funny-things.jpg"
+ 
+[right options]
+fileInput "Google key json file (required):" "Choose a file" FILE_INPUT None on None
+dropdownList "Language of the speech:" "" LANGUAGE_SYNTHESIS_INPUT en-US/it-IT/pt-PT/fr-FR/en-GB on None
+dropdownList "Voice types" "" VOICE_NAME_INPUT en-US-Wavenet-A/en-US-Wavenet-B/en-US-Wavenet-C/en-US-Wavenet-D/en-US-Wavenet-E/en-US-Wavenet-F/en-US-Wavenet-G/en-US-Wavenet-H/en-US-Wavenet-I/en-US-Wavenet-J on None
+audioInput "" "Try your voice!" AUDIO_INPUT /home/laura/teamcode/appsAway/demos/synthesis/audioicon.png on None
```
Comment (vvasco) [inline demos/funnyThingsApp/gui/gui_conf.ini:11]: We have to fix the path here

Comment (vtikha) [reply demos/funnyThingsApp/gui/gui_conf.ini:11]: @lauracavaliere  `AUDIO_INPUT`. cannot be the directory of your pc...

Comment (vtikha) [reply demos/funnyThingsApp/gui/gui_conf.ini:11]: @ilaria-carlini I would suggest that we fix this in this PR, merge then deal with all the other `gui_conf.ini` changes separately in a branch.

**Comment by AlexAntn** [comment]
LGTM

**Pull request merged**